e2e-tests/tools/daemon-util/index.js

```diff
@@ -39,6 +42,15 @@ function startNpmScript(script, color) {
 
   child.on('exit', () => {
     console.log(normal(`Finished "npm run ${script}"`));
+    // This is needed to kill the node server process on windows which is not automatically killed
+    if(platform.os.family === 'Win32') {
+      cmd.get(
+        'TASKKILL /F /IM node.exe',
+        function(err, data, stderr) {
+            console.log(err)
```

Comment:
Doesn't `stderr` get thrown when present? Is this `console.log` needed?

If it is, lets edit it to be: `console.error(error);`

Reply:
Updated